src/commands/client/resumable-query/index.test.ts

```diff
@@ -0,0 +1,85 @@
+import { afterAll, describe, expect, test } from "bun:test";
+import { QueryCommand, UpsertCommand } from "@commands/index";
+import { Index, awaitUntilIndexed, newHttpClient, randomID, range } from "@utils/test-utils";
+import { sleep } from "bun";
+
+const client = newHttpClient();
+
+describe("RESUMABLE QUERY", () => {
+	const index = new Index();
+	test("should start query successfully", async () => {
+		const { start } = await index.resumableQuery({
```

Comment:
we don't call stop on this resumable query. So the ['sources are never released'](https://github.com/upstash/vector-store/blob/master/user-api.md#end-resumable-query). What does that mean? @mdumandag

Shouldn't we stop every query we start in tests?

Reply:
If you have more than 100 (the default value) active resumable queries, you won't be able to start new ones without calling end on some of the active ones. So, yes I think we should end all the resumable queries in tests. They will be cleaned up periodically, but we better do it manually here